app/src/routes/event/[id]/+page.server.ts

```diff
@@ -19,10 +20,18 @@ export const load: PageServerLoad = async ({ params: { id }, locals: { loadQuery
   const form = await superValidate(zod(registrationSchema));
   const initial = await loadQuery<Event>(query, { id });
 
+  const eventId = (await getEvent(id)).data?.event_id;
```

Comment:
Synes denne syntaksen kan være vanskelig å lese. Pleier først å hente event og deretter sjekke om eventId eksisterer for å så bruke det videre.